CHANGELOG.rst

```diff
@@ -4,16 +4,45 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
-1.3.0-SNAPSHOT (2021-03-23)
+1.2.4 (2021-06-04)
 ---------------------------
 
 **Added**
 
 **Fixed**
 
+- App now writes the property file correctly even on failure
+- App returns exit code of 1, if an exception was detected during the sample update
+
+**Dependencies**
+
+**Deprecated**
+
+1.2.3 (2021-03-25)
```

Comment:
```suggestion
1.3.0-SNAPSHOT (2021-06-04)
--------------------------------

**Added**

**Fixed**

**Dependencies**

**Deprecated**

1.2.4 (2021-06-04)
---------------------------

**Added**

**Fixed**

- App now writes the property file correctly even on failure
- App returns exit code of 1, if an exception was detected during the sample update

**Dependencies**

**Deprecated**

1.2.3 (2021-03-25)
```
I think this should be 1.3.0-SNAPSHOT and the version should be added instead of overwritten